lang/ilias_de.lang

```diff
@@ -55,6 +55,12 @@ msg_no_connection_to_server#:#Ilias konnte keine Antwort von dem MUMIE-Server er
 grades#:#Noten
 frm_sync_lp#:#Noten-Synchronisation
 frm_sync_lp_desc#:#Wählen Sie, ob Noten für diese MumieTask automatisch mit Ilias synchronisiert werden sollen
+frm_privategradepool#:#Punkte mit anderen Kursen teilen
+frm_privategradepool_desc#:#Bestimmen Sie, ob MUMIE-Punktzahlen mit anderen Kursen geteilt werden.
+frm_privategradepool_decided#:#<b>Hinweis:</b><br> Diese Entscheidung war <b>endgültig</b> und gilt auch für alle anderen MUMIE Tasks in diesem Kurs.
```

Comment:
Please write `MUMIE-Task` instead of `MUMIE Task` in German localization. It looks weird, but that's how German works